services/postfix/exim2nix.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# Python 3.6+
+# Converts exim aliases text file to Nix
+# Usage: cat exim_aliases.txt | ./exim2nix.py > aliases.nix
```

Comment:
Why can't we just convert this once and maintain it in nix

Reply:
That's exactly what we've done. I've kept this here because I keep all my scripts and notes on mappings for the off-chance that someone needs to do it again (read: me)

Reply:
i just feel like keeping the txt file and nix file could be confusing for future admins

Reply:
I'm only keeping the python script.